bundle/manifests/saas-operator.clusterserviceversion.yaml

```diff
@@ -440,6 +440,12 @@ metadata:
                     "key": "RH_CUSTOMER_PORTAL_CLIENT_SECRET",
                     "path": "secret/data/path/system-app"
                   }
+                },
+                "realm": {
+                  "fromVault": {
```

Comment:
Do you think it's needed to store the URL in Vault? IMO, it can be a clear value.

Reply:
Good catch! That's an error, going to change it for string

Reply:
Fixed in https://github.com/3scale-ops/saas-operator/pull/229/commits/771b90ed9fdd3a8290aeb3f3fc6baa69df854be3